.ci/Jenkinsfile

```diff
@@ -384,7 +384,11 @@ def generateFunctionalTestStep(Map args = [:]){
                   }
                 } else {
                   withGoEnv(version: "${GO_VERSION}") {
-                    sh script: """.ci/scripts/functional-test.sh "${suite}" "${tags}" "${STACK_VERSION}" "${BEAT_VERSION}" """, label: "Run functional tests for ${platform}:${suite}:${tags}"
+                    // This step will help to send the APM traces to the
+                    // APM service defined by the Otel Jenkins plugin.
+                    withOtelEnv() {
```

Comment:
Is it supported on Windows? If so, L381 should be wrapped too

Reply:
Good catch, it's OS agnostic.

Reply:
https://github.com/elastic/e2e-testing/pull/1503/commits/948efc58dab7b3a25ae8c020325b522095d8cfda could be one of the approaches. IIUC, traces on Windows were not enabled, since the cmd step did run the go step itself rather than the make command.